src/Microsoft.ML.ImageAnalytics/ImageLoader.cs

```diff
@@ -3,10 +3,13 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Collections.Concurrent;
```

Comment:
>Concurrent [](start = 25, length = 10)

This and `System.Runtime.InteropServices` are not used, please remove them.
Also, `System.Diagnostics.Contracts` should not be used, on line 328 you have `Contract.Assert` instead of `Contracts.Assert`.